gpiocontrol/source/c/CMakeLists.txt

```diff
@@ -47,9 +47,11 @@ else()
 endif()
 
 # For most embedded targets binary stripping is required.
-if(${KAA_PRODUCE_BINARY})
-    add_custom_command(TARGET ${APP_NAME} POST_BUILD
-            COMMAND ${CMAKE_OBJCOPY} -O binary ${APP_NAME} ${APP_NAME}.bin)
+if("${KAA_PLATFORM}" STREQUAL "posix")
```

Comment:
oops. I've missed this disables producing binaries for all non-posix platforms. (Basically, all platforms we want binaries for.)